runtime/compiler/control/J9Options.cpp

```diff
@@ -2075,9 +2087,13 @@ J9::Options::setupJITServerOptions()
          TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "JITServer Server Mode. Port: %d. Connection Timeout %ums",
                persistentInfo->getJITServerPort(), persistentInfo->getSocketTimeout());
       else if (persistentInfo->getRemoteCompilationMode() == JITServer::CLIENT)
+         {
```

Comment:
If you added curly brackets for the else part you need to add them for the `if` part as well. That's our coding convention.